README.md

```diff
@@ -1,5 +1,27 @@
 # nested-pandas
-WORK IN PROGRESS: Efficient Pandas representation for nested associated datasets.
+Efficient Pandas representation for nested associated datasets.
+
+Nested-Pandas extends the Pandas package with tooling and support for nested 
```

Comment:
nit: Pandas often isn't capitalized. See geopandas https://github.com/geopandas/geopandas as an example of how another package references it.

Also would be nice to link to https://pandas.pydata.org/